src/app/PromptDialog.qml

```diff
@@ -30,9 +30,6 @@ ModalDialog {
     signal accept(string text)
     signal reject()
     
-    onAccept: hide()
-    onReject: hide()
-
```

Comment:
those should not be removed IMO.
Right now it works without them, because within one function only one dialog is used, and it disappears when the respective function that created the dialog object is finished (the object goes out of scope).
I've tested it with a prompt dialog and another dialog in the same function

                var confirmDialog = PopupUtils.open(Qt.resolvedUrl("ConfirmDialog.qml"), this);
                var alertDialog = PopupUtils.open(Qt.resolvedUrl("AlertDialog.qml", this))

then you can not close the confirm dialog any more by pressing OK or Cancel. Also if there is sth, that takes longer to process in the same scope, the dialog would stay open during that time, even after the user clicked OK / Cancel.
All other dialogs do have the respective `hide()` or `PopupUtils.close(dialog)` methods, too.

Reply:
Yes it was what i've used before ( hide() on accept signal in the dialog), but for some reason i had webpage freeze (even tab freeze) when doing that on big dropdowns. ( see https://lduboeuf.github.io/select.html for tests) .

Reply:
i'm going to test without hide()

Reply:
well no luck

Reply:
ok, but for the PromptDialog the old way can be kept.

Reply:
yes seems not a problem for other cases